networkx/classes/graph.py

```diff
@@ -1224,92 +1224,115 @@ def adjacency_iter(self):
         return iter(self.adj.items())
 
     def degree(self, nbunch=None, weight=None):
-        """Return the degree of a node or nodes.
+        if nbunch in self:
```

Comment:
I'll update it in the final changes :). I just wanted to get the implementation reviewed right now.